packages/core/src/transport/batch.ts

```diff
@@ -113,11 +115,11 @@ export class Batch {
 
   private willReachedBytesLimitWith(messageBytesSize: number) {
     // byte of the separator at the end of the message
-    return this.bufferBytesSize + messageBytesSize + 1 >= this.bytesLimit
+    return this.bufferBytesSize + messageBytesSize + 1 >= this.batchBytesLimit
   }
 
   private isFull() {
-    return this.bufferMessageCount === this.maxSize || this.bufferBytesSize >= this.bytesLimit
+    return this.bufferMessageCount === this.batchMessagesLimit || this.bufferBytesSize >= this.batchBytesLimit
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏should we normalize other namings as well?

- bufferMessageCount -> bufferMessagesCount 
- bufferBytesSize -> bufferBytesCount

Reply:
sure